source/build/Gdk/MainWin32.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#include "TestAppPch.h"
+#include "TestApp.h"
```

Comment:
source\build\gdk seems odd place for test code.  i haven't looked at full project layout but consider moving test app code to a test folder. eg something like source\tests\gdk or whatever makes sense.